pkg/localize/locales/en/cmd/cluster.en.toml

```diff
@@ -149,50 +142,38 @@ one = 'Connect your services to Kubernetes or OpenShift'
 
 [cluster.connect.cmd.longDescription]
 one = '''
-Connect your application services to your Kubernetes or OpenShift cluster.
-The kubeconfig file is used to connect to the cluster and identify the context.
-Alternatively, you can use the KUBECONFIG environment variable to read your configuration.
-
-NOTE: This command requires the RHOAS operator to be running on your cluster.
-
-This command will:
+Connect your application services to an Kubernetes or OpenShift cluster.
 
-1. Create a service account and mount it as a secret into your cluster.
-If the cluster has a service account, it will be refreshed.
+Note: Before you can connect your application service to OpenShift, you need to install the RHOAS Operator in your OpenShift cluster. For installation instructions, see https://github.com/redhat-developer/app-services-guides/tree/main/service-discovery#installing-the-rhoas-operator-on-openshift.
 
-2. Create a Kafka Request object that can be used to create a ServiceBinding object using
-the Service Binding operator (https://github.com/redhat-developer/service-binding-operator).
+When you connect an application service to a Kubernetes or OpenShift cluster, this command uses the kubeconfig file (or the KUBECONFIG environment variable) to connect to the cluster and identify the context.
 
-NOTE: Created service account will need to be given permissions to specified service
-For example for Kafka service you should execute the following command to grant access to the service account
+After identifying the context, the RHOAS Operator creates a service account and mounts it as a secret in your cluster. If your cluster has a service account already, the service account is refreshed.
 
-  $ rhoas kafka acl grant-access --producer --consumer --service-account your-sa --topic all --group all
+Finally, the RHOAS Operator creates a Kafka Request Object, which the Service Binding Operator (https://github.com/redhat-developer/service-binding-operator) uses to create a "ServiceBinding" object.
 
-Similarly for service registry you should execute the following command:
-
-  $ rhoas service-registry role add --role=DEVELOPER --service-account your-sa
+After running this command, you need to grant access for the service account that was created by the RHOAS Operator to access your application service:
 
+$ rhoas kafka acl grant-access --producer --consumer --service-account your-sa --topic all --group all
```

Comment:
We are missing distiction between service registry and kafka access rights (This was specifically required depending on the type of the service we have.

Can we restore content starting from 

`NOTE: Created service account will need to be given permissions to specified service

Reply:
I think I captured the requested portion. Here's how the full long description now reads. Let me know if this isn't correct:

```
Connect your application services to an Kubernetes or OpenShift cluster.

Note: Before you can connect your application service to OpenShift, you need to install the RHOAS Operator in your OpenShift cluster. For installation instructions, see https://github.com/redhat-developer/app-services-guides/tree/main/service-discovery#installing-the-rhoas-operator-on-openshift.

When you connect an application service to a Kubernetes or OpenShift cluster, this command uses the kubeconfig file (or the KUBECONFIG environment variable) to connect to the cluster and identify the context.

After identifying the context, the RHOAS Operator creates a service account and mounts it as a secret in your cluster. If your cluster has a service account already, the service account is refreshed.

Finally, the RHOAS Operator creates a Kafka Request Object, which the Service Binding Operator (https://github.com/redhat-developer/service-binding-operator) uses to create a "ServiceBinding" object.

After running this command, you need to grant access for the service account that was created by the RHOAS Operator to access your application service. For the Kafka application service, enter the following command:

  $ rhoas kafka acl grant-access --producer --consumer --service-account your-sa --topic all --group all

For the Service Registry application service, enter this command:

  $ rhoas service-registry role add --role=DEVELOPER --service-account your-sa
```